<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphwindow</class>
 <widget class="QDialog" name="graphwindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>662</width>
    <height>608</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>662</width>
    <height>608</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton_build">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(255, 255, 255);
  color: rgb(0, 0, 0);
  border: 1px solid gray;
  border-radius: 5px;
}
QPushButton:pressed {
    background-color: rgb(82, 148, 255);
}</string>
   </property>
   <property name="text">
    <string>build the graph</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_function">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>80</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_y">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>function:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_min">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>x min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_max">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>x max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_min">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>y min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_max">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>y max</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="x_min">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="x_max">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="y_min">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>-10.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="y_max">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-1000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphics_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>645</width>
     <height>485</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>0.000000000000000</width>
     <height>0.000000000000000</height>
    </rectf>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
